arm-advisor/2017-03-31/swagger/advisor.json

```diff
@@ -0,0 +1,679 @@
+						"required": true,
+						"type": "string"
+					},
+					{
+						"name": "recommendationId",
+						"in": "path",
+						"description": "The recommendation ID.",
+						"required": true,
+						"type": "string"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResourceRecommendationBase"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"GetRecommendationDetail": {
+						"$ref": "../examples/GetRecommendationDetail.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/suppressions": {
+			"get": {
+				"tags": [
+					"Suppressions"
+				],
+				"description": "Retrieves the list of snoozed or dismissed suppressions for a subscription. The snoozed or dismissed attribute of a recommendation is referred to as a suppression.",
+				"operationId": "Suppressions_List",
+				"parameters": [
+					{
+						"$ref": "#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK.",
+						"schema": {
+							"type": "array",
+							"items": {
+								"$ref": "#/definitions/SuppressionContract"
+							}
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"ListSuppressions": {
+						"$ref": "../examples/ListSuppressions.json"
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"ResourceRecommendationBaseListResult": {
+			"description": "The list of Advisor recommendations.",
+			"type": "object",
+			"properties": {
+				"nextLink": {
+					"description": "The link used to get the next page of recommendations.",
+					"type": "string"
+				},
+				"value": {
+					"description": "The list of recommendations.",
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/ResourceRecommendationBase"
+					}
+				}
+			}
+		},
+		"ResourceRecommendationBase": {
+			"description": "Advisor Recommendation.",
+			"type": "object",
+			"properties": {
+				"id": {
+					"description": "The fully qualified recommendation ID, for example /subscriptions/subscriptionId/resourceGroups/resourceGroup1/providers/Microsoft.ClassicCompute/virtualMachines/vm1/providers/Microsoft.Advisor/recommendations/recommendationGUID.",
+					"type": "string"
+				},
+				"name": {
+					"description": "The name of recommendation.",
+					"type": "string"
+				},
+				"properties": {
+					"$ref": "#/definitions/RecommendationProperties",
+					"description": "The properties of the recommendation.",
+					"x-ms-client-flatten": true
+				},
+				"suppressionIds": {
+					"description": "The list of snoozed and dismissed rules for the recommendation.",
+					"type": "array",
+					"items": {
+						"format": "uuid",
```

Comment:
Hank and I had a conversation on this. The design of this API will be revisited after GA. There may not be a need for suppressionIds. But lets not block GA due to this. This is read-only for now and service generates it. So user does not have to pass in the GUID.